### PR TITLE
Change Request Base Fixes

### PR DESCRIPTION
**Pull request by gonzalesedwin1123**
Remove domain attribute linked with the OCA dms demo data.

### CODE REVIEW

File: spp_change_request/views/dms_file_view.xml [hunk 1]
```diff
@@ -27,7 +27,6 @@
                                 options="{'no_open': True,'no_create': True,'no_edit':True}"
                                 required="1"
                                 readonly="context.get('category_readonly')"
-                                domain="[('parent_id', 'child_of', %(dms.category_01_demo)d)]"
```
Comment (nhatnm0612) [inline spp_change_request/views/dms_file_view.xml:30]: @gonzalesedwin1123 pls don't remove this line. It is up to the request from the issue 186858

Comment (gonzalesedwin1123) [reply spp_change_request/views/dms_file_view.xml:30]: OK @nhatnm0612, thanks for the info but I will just add a category for this. It is generating an error in the installation  of the spp_change_request if the "Install demo data" is off.

Comment (jeremi) [reply spp_change_request/views/dms_file_view.xml:30]: @gonzalesedwin1123 is the category added? Can we merge this?

Comment (gonzalesedwin1123) [reply spp_change_request/views/dms_file_view.xml:30]: @jeremi the category is now in the latest commit. It can be merged now.

**Review by nhatnm0612 — APPROVED** [review]

**Review by jeremi — APPROVED** [review]

**Pull request merged**